src/test/kotlin/org/opensearch/replication/MultiClusterRestTestCase.kt

```diff
@@ -664,6 +659,16 @@ abstract class MultiClusterRestTestCase : OpenSearchTestCase() {
         return integTestRemote.equals("true")
     }
 
+    protected fun isMultiNodeClusterConfiguration(leaderCluster: String, followerCluster: String): Boolean{
+        val systemProperties = BootstrapInfo.getSystemProperties()
+        val totalLeaderNodes = systemProperties.get("tests.cluster.${leaderCluster}.total_nodes") as String?
+        val totalFollowerNodes = systemProperties.get("tests.cluster.${followerCluster}.total_nodes") as String?
+        if(totalLeaderNodes != null && totalFollowerNodes !=null && totalLeaderNodes < "2" && totalFollowerNodes < "2" ) {
+            return false
+        }
+        return true
```

Comment:
Why `true` is the default behaviour here? If the system property is not getting stuck, we'll end up not skipping the tests and thus failing.

Reply:
If integ tests are run locally, we are always setting the property and same with integTest Remote

Reply:
then let's remove the null check

Reply:
and add an assertion instead